superset-frontend/src/dashboard/actions/dashboardInfo.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Dispatch } from 'redux';
+import { makeApi } from '@superset-ui/core';
+import { ChartConfiguration, DashboardInfo } from '../reducers/types';
+
+export const DASHBOARD_INFO_UPDATED = 'DASHBOARD_INFO_UPDATED';
+
+// updates partially changed dashboard info
+export function dashboardInfoChanged(newInfo: { metadata: any }) {
+  return { type: DASHBOARD_INFO_UPDATED, newInfo };
+}
+export const SET_CHART_CONFIG_BEGIN = 'SET_CHART_CONFIG_BEGIN';
+export interface SetChartConfigBegin {
+  type: typeof SET_CHART_CONFIG_BEGIN;
+  filterConfig: ChartConfiguration;
```

Comment:
nit: it would be nice if we are consistent with naming: `filterConfig` vs `filterConfiguration`. Either is fine, but it seems we already have fields with `_configuration` in the dashboard metadata, so perhaps that one is preferable. Also, shouldn't this be `chartConfig`/`chartConfiguration`?